manifests/w/Wasmer/Wasmer/4.2.0/Wasmer.Wasmer.installer.yaml

```
@@ -1,25 +1,25 @@
-# Created with YamlCreate.ps1 v2.2.10 $debug=QUSU.CRLF.7-3-6.Win32NT
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
-
-PackageIdentifier: Wasmer.Wasmer
-PackageVersion: 4.2.0
-InstallerType: inno
-Scope: machine
-UpgradeBehavior: install
-Commands:
-- wasmer
-- wasmer-headless
-- wax
-FileExtensions:
-- wasm
-Dependencies:
-  PackageDependencies:
-  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
-ReleaseDate: 2023-09-05
-Installers:
-- Architecture: x64
-  InstallerUrl: https://github.com/wasmerio/wasmer/releases/download/v4.2.0/wasmer-windows.exe
-  InstallerSha256: A6D631C6E288A2CC492AC950830BC01D1D18B598973C722AAA1D9EF6FF69F754
-  ProductCode: Wasmer_is1
-ManifestType: installer
-ManifestVersion: 1.5.0
+# Created with WinGet Automation using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Wasmer.Wasmer
+PackageVersion: 4.2.0
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Commands:
+- wasmer
+- wasmer-headless
+- wax
+FileExtensions:
+- wasm
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+ProductCode: Wasmer_is1
+ReleaseDate: 2023-09-05
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
Guessing this is 64-bit.